pay/example/lib/main.dart

```diff
@@ -44,6 +45,8 @@ class PayMaterialApp extends StatelessWidget {
         const Locale('es', ''),
         const Locale('de', ''),
       ],
+      theme: ThemeData.light(),
+      darkTheme: ThemeData.dark(),
```

Comment:
I suggest to keep only the regular `theme` to avoid adding too much config substance or opinion to a demo application.

Reply:
Sure 👍